packages/shared/src/UI/components/CollectionDetailCard/index.tsx

```diff
@@ -0,0 +1,284 @@
+                                    loadingFailImage: classes.loadingFailImage,
+                                    wrapper: classes.img,
+                                    iframe: classes.img,
+                                }}
+                            />
+                        </Card>
+                    </Box>
+
+                    <Typography fontSize="16px" fontWeight={700} marginTop="38px">
+                        {title}
+                    </Typography>
+                    <div className={classes.icons}> {icons}</div>
+
+                    <Link rel="noopener noreferrer" target="_blank" href={referenceUrl} className={classes.link}>
+                        {referenceUrl}
+                    </Link>
+                    {date && (
+                        <Typography fontSize="14px" fontWeight={400} marginTop="12px">
+                            {date}
+                        </Typography>
+                    )}
+                    {location && (
+                        <Typography fontSize="14px" fontWeight={400} marginTop="8px">
+                            <span className={classes.themeColor}>@</span>
+                            {location}
+                        </Typography>
+                    )}
+                    <Typography fontSize="16px" fontWeight={700} marginTop="16px">
+                        {t.description()}
+                    </Typography>
+                    <div style={{ display: '-webkit-box' }} className={classes.threeLine}>
+                        {description}
+                    </div>
+
+                    {contributions ? (
+                        <Typography fontSize="16px" fontWeight={700} marginTop="16px">
+                            {t.contributions()}
+                        </Typography>
+                    ) : null}
+                    {contributions ? (
+                        <Typography fontSize="16px" fontWeight={700} marginBottom="16px">
+                            {contributions?.length ?? 0}
+                        </Typography>
+                    ) : null}
+                    {contributions?.map((contribution) => (
+                        <div key={contribution.txHash} className={classes.txItem}>
+                            <Typography className={classes.donationAmount}>
+                                {contribution.formatedAmount} {contribution.symbol}
+                            </Typography>
+                            <div className={classes.dayBox}>
+                                {differenceInCalendarDays(new Date(), new Date(Number(contribution.timeStamp) * 1000))}{' '}
+                                {t.days()}{' '}
+                                {differenceInCalendarHours(
+                                    new Date(),
+                                    new Date(Number(contribution.timeStamp) * 1000),
+                                ) % 24}{' '}
+                                {t.hours()} {t.ago()}
```

Comment:
```suggestion
                                {differenceInCalendarDays(Date.now(), Number(contribution.timeStamp) * 1000)} {t.days()}{' '}
                                {differenceInCalendarHours(Date.now(), Number(contribution.timeStamp) * 1000) % 24}{' '}
                                {t.hours()} {t.ago()}

```

BTW, should we consider singular?